recipes/starshot_admin_theme/recipe.yml

```diff
@@ -11,6 +11,11 @@ config:
     gin: '*'
     navigation: '*'
   actions:
+    gin.settings:
+      simple_config_update:
+        high_contrast_mode: true
+        accent_color: ''
+        focus_color: ''
```

Comment:
One question - why are the accent and focus colors removed here?

Reply:
They aren't set until the high contrast is enabled, when the setting is switched on, those values are set to an empty string and could be changed if necessary, the defaults are an improvement as they are, but I will consult others to see if a better setting could be used.

Reply:
Bizarrely, I double-checked where this came from by installing starshot and exporting the config.

I then switched on the high contrast toggle and exported again to see the diff. The accent and focus colours were not in the original export, but are added when the high contrast is switched on. There are other changes, but mostly just moving config up and down in the config file. I'll remove them as they seem to be set not changed.